helpers/auth/authentication.py

```diff
@@ -63,7 +63,38 @@ def decode_token(self):
                 'Invalid token. Please Provide a valid token!'
             }), 401
 
-    def save_user(self):
+    def andela_api_call(self, email, *expected_args):
+        """accepts a users email and makes a call to the Andela api
+            and returns the users information
+        """
+        try:
+            headers = {"Authorization": 'Bearer ' + self.get_token()}
+            if(os.getenv('APP_SETTINGS') == "testing"):
+                with open('users.json', 'r') as f:
+                    response = json.load(f)
+            else:  # pragma: no cover
+                data = requests.get(
+                    api_url + "users?email=%s"
+                    % email, headers=headers)
+                response = json.loads(data.content.decode("utf-8"))
+
+            return response
+        except requests.exceptions.ConnectionError:
+            message = "Failed internet connection"
+            status = 408
+            handle_http_error(message, status, expected_args)
+
+        if 'error' in response:  # pragma: no cover
+            message = response['error']
+            status = 401
+            if(message == "invalid token"):
+                handle_http_error(
+                    "You don't have a valid token \
```

Comment:
You can try reduce the spacing on this message as it appears this way

![image](https://user-images.githubusercontent.com/41931013/63019081-1901b600-bea3-11e9-878a-f2cc7894c3f8.png)

Reply:
Fixed. Still working on the code coverage issue.